website/src/components/Buttons/3p.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button, ButtonProps } from "@chakra-ui/react";
+
+export const SocialLoginButton = ({ children, ...props }: ButtonProps) => {
```

Comment:
I think this PR needs some guidance from @k-nearest-neighbor.  Right now this new component doesn't seem to add any additional features ontop of `Button`.  I'd like @k-nearest-neighbor to give some clarification on the re-structuring he had in mind.